packages/wdio-sauce-service/src/types.ts

```diff
@@ -24,13 +24,21 @@ export interface SauceServiceConfig {
      * @default false
      */
     sauceConnect?: boolean
+
     /**
      * Apply Sauce Connect options (e.g. to change port number or logFile settings). See this
      * list for more information: https://github.com/bermi/sauce-connect-launcher#advanced-usage
      *
      * @default {}
      */
     sauceConnectOpts?: SauceConnectOptions
+
+    /**
+     * Upload WebdriverIO logs to the Sauce Labs platform.
+     * @default false
```

Comment:
Does this really slow down the execution if we just enable it by default? I think it would be really helpful to always have this information in Sauce

Reply:
I am not too much concerned about execution time but rather about whether or not people actually look into the data. We can have it default to `true` and have a dashboard for this to check if people actually use it or not.